ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/toggleablelink/v1/toggleablelink/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,108 @@
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <placeholder
+                                              jcr:primaryType="nt:unstructured"
+                                              sling:hideResource="{Boolean}true" />
+                                            <type
+                                              jcr:primaryType="nt:unstructured"
+                                              sling:hideResource="{Boolean}true" />
+                                            <value
+                                              jcr:primaryType="nt:unstructured"
+                                              sling:hideResource="{Boolean}true" />
+                                            <alignment
+                                              jcr:primaryType="nt:unstructured"
+                                              sling:hideResource="{Boolean}true" />
+                                            <enums jcr:primaryType="nt:unstructured"
+                                              sling:resourceType="granite/ui/components/coral/foundation/container">
+                                                <items jcr:primaryType="nt:unstructured">
+                                                    <enumOptions jcr:primaryType="nt:unstructured"
+                                                      sling:orderBefore="bindref"
+                                                      sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
+                                                      fieldDescription="Enter the display text with link."
+                                                      fieldLabel="Links">
+                                                        <field jcr:primaryType="nt:unstructured"
+                                                          sling:resourceType="granite/ui/components/coral/foundation/container"
+                                                          name="./enum">
+                                                            <items jcr:primaryType="nt:unstructured">
+                                                                <enum jcr:primaryType="nt:unstructured"
+                                                                  sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                                  granite:class="cmp-adaptiveform-base__enum"
+                                                                  name="./enum"
+                                                                  fieldLabel="Link"
+                                                                  fieldDescription="Enter the link."
+                                                                  required="{Boolean}true"/>
+                                                                <enumNames
```

Comment:
Can you implement the rich text which was recently added by @im-shiv, its missing here ? Please also add test cases for the same.

Reply:
This is already done 
https://github.com/adobe/aem-core-forms-components/blob/07e8c84dd13ff01dcd50bed08c242981f19121ae/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/toggleablelink/v1/toggleablelink/widget.html#L30